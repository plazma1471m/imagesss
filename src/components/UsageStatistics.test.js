import React from 'react'
import renderer from 'react-test-renderer'
import UsageStatistic from './UsageStatistic'

it('renders statistics with default values correctly', () => {
  const tree = renderer.create(<UsageStatistic />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders statistics with prop values correctly', () => {
  const tree = renderer
    .create(<UsageStatistic filesProcessed={3} bytesProcessed={2048} bytesSaved={1024} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
