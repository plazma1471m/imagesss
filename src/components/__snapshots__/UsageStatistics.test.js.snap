// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders statistics with default values correctly 1`] = `
<div
  className="ui attached segment"
>
  <div
    className="ui blue mini two statistics"
  >
    <div
      className="ui statistic"
    >
      <div
        className="value"
      >
        0
      </div>
      <div
        className="label"
      >
        Images processed
      </div>
    </div>
    <div
      className="ui statistic"
    >
      <div
        className="value"
      >
        0 B
         (
        0%
        )
      </div>
      <div
        className="label"
      >
        Data saved
      </div>
    </div>
  </div>
</div>
`;

exports[`renders statistics with prop values correctly 1`] = `
<div
  className="ui attached segment"
>
  <div
    className="ui blue mini two statistics"
  >
    <div
      className="ui statistic"
    >
      <div
        className="value"
      >
        3
      </div>
      <div
        className="label"
      >
        Images processed
      </div>
    </div>
    <div
      className="ui statistic"
    >
      <div
        className="value"
      >
        1.02 kB
         (
        50%
        )
      </div>
      <div
        className="label"
      >
        Data saved
      </div>
    </div>
  </div>
</div>
`;
